### ISSUE TITLE
Linking Fails on AURGAViewer.a; Running the binary directly gives a Segmenation fault (Kubuntu 22.04)

### ISSUE THREAD

**Issue by colinpickup** [aurgatech/linux-binaries#3]
I am following the instructions, but getting the following code when I try and link the file:
```
/usr/bin/ld: AURGAViewer.a(Main.o): relocation R_X86_64_32 against `.bss' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: failed to set dynamic section sizes: bad value
collect2: error: ld returned 1 exit status
```
From my understanding, the recompilation with -fPIE would need to be done when the AURGAViewer.a filewas generated, not when I am linking.

If I try and run the binaries included in the distribution, it just outputs "Segmentation fault".

Has anybody managed to get past this?

**Comment by xperroni** [comment]
I have run into the same issue. My environment is also Kubuntu 22.04. I followed the steps for building FFMPEG and tried linking the binary in both the `OpenGLES2/` and `OpenGL3/` directories. In both cases I got the same error message:

```
/usr/bin/ld: AURGAViewer.a(Main.o): relocation R_X86_64_32 against `.bss' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: failed to set dynamic section sizes: bad value
collect2: error: ld returned 1 exit status
```

**Comment by BohdanTkachenko** [comment]
Same issue here with Pop!_OS. As a workaround I just launched Windows version through Wine.

**Comment by stuckj** [comment]
Yep, fails in ubuntu mantic (23.10) as well. I'm guessing the devs don't really pay attention to Linux considering they their install instructions involved linking static libraries. :-P

**Comment by stuckj** [comment]
Based on the first [issue](https://github.com/aurgatech/linux-binaries/issues/1) it sounds like just installing the windows version with wine is probably the way to go.

**Comment by stuckj** [comment]
Verified. Wine is a much better solution. Download the Windows x86 installer for win8+ and just run `wine AURGAViewer_Installer_x64_v1.0.9.0.exe`.
